jujugui/static/gui/src/app/components/deployment/input/_input.scss

```diff
@@ -12,8 +12,6 @@
         width: calc(100% - 20px);
         height: 20px;
         margin-bottom: 20px;
-        padding-top: 25px;
-        padding-bottom: 7px;
```

Comment:
I actually overwrite the padding entirely in the _user-profile.scss so I've reverted this change. Good catch thx